simple_history/tests/tests/test_utils.py

```diff
@@ -423,6 +425,15 @@ def test_bulk_update_history_with_batch_size(self):
         self.assertEqual(Poll.objects.count(), 5)
         self.assertEqual(Poll.history.filter(history_type="~").count(), 5)
 
+    @skipUnless(VERSION >= (4, 0), "Requires Django 4.0 or above")
+    def test_bulk_update_history_row_updated(self):
```

Comment:
A name similar to this will probably make the purpose of the test clearer to readers:
```suggestion
    def test_bulk_update_with_history_returns_rows_updated(self):
```